tests/plugins/autogrow/manual/4372.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 4.15.1, bug, 7372
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, autogrow
+
+
+1. Focus inside editable window.
+2. Add or remove extra lines to force editor resize.
+3. Observe editor width.
+
+#### Expected result:
+The editor should have constant width. Only height is changing according to vertical amount of content.
+
+#### Unexpected result:
+With each vertical resize editor is shrinking for about margins + paddings.
```

Comment:
```suggestion
With each vertical resize editor is shrinking for about 2 pixels.
```

Reply:
That was my first thought. Later it comes to my mind (I may be over-obsessed) if the default margin and padding will change, those `2px` will be invalid. Anyway, I apply suggestion :)